README.md

```diff
@@ -52,6 +52,26 @@ make VERSION=0.3.0 push
 make clean
 ```
 
+On Windows, using the tools described below, use the command:
+
+"C:\Program Files\git\bin\bash" -c "make"
```

Comment:
Powershell technique added in https://github.com/drud/build-tools/pull/37/commits/88ec0ae119f836ce767c8291e295168b24b0c05a - Maybe someday I'll know more of these things!